Mathlib/Data/List/Chain.lean

```diff
@@ -440,3 +440,50 @@ theorem relationReflTransGen_of_exists_chain (l : List α) (hl₁ : Chain r a l)
 #align list.relation_refl_trans_gen_of_exists_chain List.relationReflTransGen_of_exists_chain
 
 end List
+
+
+/-! In this section, we consider the type of `r`-decreasing chains (`List.Chain' (flip r)`)
+  equipped with lexicographic order `List.Lex r`. -/
+
+variable {α : Type*} {r : α → α → Prop}
```

Comment:
What do you think about introducing an abbreviation for the repeating terms appearing below:
```suggestion
variable {α : Type*} (r : α → α → Prop)

abbrev List.chains := { l : List α // l.Chain' (flip r) }

abbrev List.lex_chains : List.chains r → List.chains r → Prop := fun l m ↦ List.Lex r l.val m.val

variable {r}
```
(and using them in the three statements below)

Reply:
Good idea!